gcloud-java-compute/src/test/java/com/google/gcloud/compute/it/ITComputeTest.java

```diff
@@ -447,4 +446,190 @@ public void testListZonesWithFilter() {
     assertEquals(ZONE, zoneIterator.next().zoneId().zone());
     assertFalse(zoneIterator.hasNext());
   }
+
+  @Test
+  public void testListGlobalOperations() {
```

Comment:
Also, has gcloud-devel been populated with some operations just so these listing functions run at least one iteration of the loops?